registry/nacos/service_discovery.go

```diff
@@ -218,14 +218,15 @@ func (n *nacosServiceDiscovery) AddListener(listener registry.ServiceInstancesCh
 	for _, t := range listener.GetServiceNames().Values() {
 		serviceName := t.(string)
 		err := n.namingClient.Client().Subscribe(&vo.SubscribeParam{
-			ServiceName: serviceName,
 			GroupName:   n.group,
+			ServiceName: serviceName,
 			SubscribeCallback: func(services []model.SubscribeService, err error) {
 				if err != nil {
 					logger.Errorf("Could not handle the subscribe notification because the err is not nil."+
 						" service name: %s, err: %v", serviceName, err)
 				}
 				instances := make([]registry.ServiceInstance, 0, len(services))
+				logger.Infof("%v", services)
```

Comment:
what is  the meaning of this log? make it more clearly

Reply:
test case which forget to be removed, sorry for my mistake